app/models/event.rb

```diff
@@ -18,26 +18,27 @@ class Event < ActiveRecord::Base
   # Returns whether all application_letters are classified or not
   #
   # @param none
-  # @return [Boolean] if status of all application_letters is not equal 2
-  def applicationsClassified?
+  # @return [Boolean] if status of all application_letters is not pending
+  def applications_classified?
     application_letters.all? { |application_letter| application_letter.status != 'pending' }
   end
 
   # Returns a string of all email addresses of rejected applications
   #
   # @param none
   # @return [String] Concatenation of all email addresses of rejected applications, seperated by ','
-  def compute_rejected_applications_emails
-    rejected_applications = application_letters.select { |application_letter| application_letter.status == 'rejected' }
+  def rejected_applications_emails
+    #rejected_applications = application_letters.select { |application_letter| application_letter.status == 'rejected' }
+    rejected_applications = application_letters.where(status: ApplicationLetter.statuses[:rejected])
```

Comment:
just for my understanding, is application_letters.x() the same model access like ApplicationLetter.x()?